baselines/compute/test/gapic_addresses_v1.ts.baseline

```diff
@@ -343,7 +343,6 @@ describe('v1.AddressesClient', () => {
                 .getCall(0).calledWith(request, expectedOptions, undefined));
         });
     });
-
```

Comment:
Maybe it's better to put this empty line back if not too hard?

Reply:
Agree.